src/main/java/fr/insalyon/creatis/gasw/execution/GaswSubmit.java

```diff
@@ -70,6 +74,10 @@ public GaswSubmit(GaswInput gaswInput, GaswMinorStatusServiceGenerator minorStat
         if (GaswConfiguration.getInstance().isFailOverEnabled()) {
             FailOver.getInstance().addData(gaswInput.getDownloads());
         }
+        System.out.println("moteurlite status: " + gaswInput.getMoteurliteStatus());
+        if (gaswInput.getMoteurliteStatus() != null && gaswInput.getMoteurliteStatus() == true) {
+            moteurliteStatus = true;
+        }
```

Comment:
`gaswInput.getMoteurliteStatus()` should not be null, you can use native `boolean` type

Reply:
Actually, this is not necessary to save `moteurLiteStatus` in a field as the information is present in `this.gaswInput`.

Reply:
The moteurliteStatus field is removed.
The check for gaswInput.getMoteurliteStatus() uses the native boolean type, removing the need to check for null.
The moteurliteStatus is checked directly from the gaswInput object where necessary.